src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -58,6 +58,30 @@ public static void setField(
         }
   }
 
+  /**
+   * Changes a given static field of a class via reflection, bypassing the
+   * private modifier.
+   *
+   * @param <T> type parameter of the class
+   * @param c the <code>Class</code> of the static field to set
+   * @param fieldName a <code>String</code> the name of the field to change
+   * @param newVal an <code>Object</code> the new value for the field
+   */
+  public static <T> void setStaticField(
```

Comment:
You should add some tests :stuck_out_tongue:

Reply:
I knooow :sweat:

Reply:
I'm trying to convince platform to act nicely :)

Reply:
I added locally generated tests for the new method. Platform takes a lot of time on this file for some reason...